vips/vip-Gateway/bscmainnet.ts

```diff
@@ -0,0 +1,247 @@
+    address: "0xfd5840cd36d94d7229439859c0112a4185bc0255",
+    holder: "0xee5b5b923ffce93a870b3104b7ca09c3db80047a",
+  },
+  {
+    name: "vWBETH",
+    address: "0x6cfdec747f37daf3b87a35a1d9c8ad3063a1a8a0",
+    holder: "0xf977814e90da44bfa03b6295a0616a897441acec",
+  },
+  {
+    name: "vXRP",
+    address: "0xb248a295732e0225acd3337607cc01068e3b9c10",
+    holder: "0x5a52e96bacdabb82fd05763e25335261b270efcb",
+  },
+  {
+    name: "vXVS",
+    address: "0x151b1e2635a717bcdc836ecd6fbb62b674fe3e1d",
+    holder: "0xfd36e2c2a6789db23113685031d7f16329158384",
+  },
+];
+
+export const vipGateway = () => {
+  const meta = {
+    version: "v2",
+    title:
+      "VIP-Gateway Update VToken and comptroller implementation in IL and Core Pool and introduces the NativeTokenGateway contract",
+    description: `
+    This VIP does the following:
+    1. Updates the implementation of all VTokens and Comptroller market facet in Core Pool
+    2. Updates the implementation of VTokens and Comptrollers in IL
+    3. Accepts the ownership of the NativeTokenGateway contract
+    4. Gives call permissions to Timelocks for seizeVenus function
+    5. Executes seizeVenus
+    6. Sets XVS and vXVS address in unitroller`,
+    forDescription: "I agree that Venus Protocol should proceed with this proposal",
+    againstDescription: "I do not think that Venus Protocol should proceed with this proposal",
+    abstainDescription: "I am indifferent to whether Venus Protocol proceeds or not",
+  };
+
+  return makeProposal(
+    [
+      {
+        target: UNITROLLER,
+        signature: "_setPendingImplementation(address)",
+        params: [DIAMOND],
+      },
+      {
+        target: DIAMOND,
+        signature: "_become(address)",
+        params: [UNITROLLER],
+      },
+      {
+        target: UNITROLLER,
+        signature: "diamondCut((address,uint8,bytes4[])[])",
+        params: [params],
+      },
+      ...CORE_MARKETS.map(vToken => {
+        return {
+          target: vToken.address,
+          signature: "_setImplementation(address,bool,bytes)",
+          params: [NEW_VBEP20_DELEGATE_IMPL, false, "0x"],
+        };
+      }),
+      {
+        target: COMPTROLLER_BEACON,
+        signature: "upgradeTo(address)",
+        params: [NEW_COMPTROLLER_IMPLEMENTATION],
+      },
+      {
+        target: VTOKEN_BEACON,
+        signature: "upgradeTo(address)",
+        params: [NEW_VTOKEN_IMPLEMENTATION],
+      },
+      {
+        target: NATIVE_TOKEN_GATEWAY,
+        signature: "acceptOwnership()",
+        params: [],
+      },
+
+      ...[NORMAL_TIMELOCK, FAST_TRACK_TIMELOCK, CRITICAL_TIMELOCK].map((timelock: string) => ({
+        target: ACM,
+        signature: "giveCallPermission(address,string,address)",
+        params: [UNITROLLER, "seizeVenus(address[],address)", timelock],
+      })),
+
+      {
+        target: UNITROLLER,
+        signature: "seizeVenus(address[],address)",
+        params: [accounts, UNITROLLER],
+      },
```

Comment:
`seizeVenus` assumes `xvs` is set. Shouldn't we invoke this command after setting the XVS (and the vXVS) tokens?

Reply:
Done https://github.com/VenusProtocol/vips/pull/195/commits/e1697b64705e249c2cfc3fc4f805be9ac5930af5